packages/create-app/templates/default-app/app-config.yaml.hbs

```diff
@@ -106,3 +106,6 @@ catalog:
 
 kubernetes:
   # see https://backstage.io/docs/features/kubernetes/configuration for kubernetes configuration options
+
+permission:
+  enabled: true
```

Comment:
This will enable the permissions framework by default correct? should we add some flavour text here as a comment something like:

This enabled the Backstage Permission Framework see: link-to-docs you can set this to false if you do not want to use the permission framework.

Reply:
Sure, that makes sense to me. We do ship a default allow all policy so this doesn't do much but it's one less bit of code/config that needs to be added.